scripts/deploy-elasticbeanstalk.sh

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env bash
+
+# https://google.github.io/styleguide/shell.xml#Naming_Conventions
+
+readonly BUILD="locksmith-$(uuidgen)"
```

Comment:
Would it make sense to use `$TRAVIS_BUILD_ID` for consistency?

Reply:
good call, I was testing this locally first but that makes way more sense 👍